pkgs/development/tools/misc/blackmagic/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, lib, fetchFromGitHub
+, gcc-arm-embedded, bash, libftdi
+, python, pythonPackages
+}:
+
+with lib;
+
+stdenv.mkDerivation rec {
+  name = "blackmagic-${version}";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "blacksphere";
+    repo = "blackmagic";
+    rev = "d3a8f27fdbf952194e8fc5ce9b2fc9bcef7c545c";
+    sha256 = "0c3l7cfqag3g7zrfn4mmikkx7076hb1r856ybhhdh0f6zji2j6jx";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = [
+    gcc-arm-embedded
+    libftdi
+    python
+    pythonPackages.intelhex
+  ];
+
+  # Prevent calling out to `git' to generate a version number:
+  postPatch = ''
+    substituteInPlace src/Makefile \
+      --replace '`git describe --always --dirty`' '${version}'
+  '';
+
+  buildPhase = "${bash}/bin/bash ${./helper.sh}";
```

Comment:
No, it's not.  But isn't bash the shell used for all the phases anyways?  What's a better option?